prosessering-core/src/test/kotlin/no/nav/familie/prosessering/rest/TaskControllerIntegrasjonTest.kt

```diff
@@ -73,6 +73,24 @@ internal class TaskControllerIntegrasjonTest : IntegrationRunnerTest() {
         assertThat(response.body.data).isEqualTo(2)
     }
 
+    @Test
+    fun `skal finne riktig antall tasker som har feilet og ligger til manuell oppfølging`() {
+        val feiletTask = Task(type = TaskStep2.TASK_2, payload = "{'a'='1'}", status = Status.FEILET)
+        val feiletTask2 = Task(type = TaskStep2.TASK_2, payload = "{'a'='1'}", status = Status.FEILET)
+        val feiletTask3 = Task(type = TaskStep2.TASK_2, payload = "{'a'='1'}", status = Status.FEILET)
```

Comment:
Legg gjerne inn en task med en annen statustype for å verifisere at den blir filtrert "vekk" :)